app/script_variant.py

```diff
@@ -44,6 +44,18 @@ def has_kanji(word):
     return re.match(ur'([\u3400-\u9fff])', word)
 
 
+def find_word_in_dict(word):
```

Comment:
This function also romanizes, not just "find". Also the function name should contain "japanese" because it uses Japanese dictionary. How about e.g., "romanize_single_japanese_word_by_name_dict"? Please also add a comment for the function.